src/main/java/graphql/execution/nextgen/ExecutionStrategyUtil.java

```diff
@@ -6,8 +6,6 @@
 import graphql.execution.ExecutionStepInfo;
 import graphql.execution.ExecutionStepInfoFactory;
 import graphql.execution.FetchedValue;
-import graphql.execution.FieldCollector;
-import graphql.execution.FieldCollectorParameters;
 import graphql.execution.MergedField;
```

Comment:
Use `graphql.execution.nextgen.FieldCollector` and `graphql.execution.nextgen.FieldCollectorParameters` which keep same before this pr.